src/pages/GameDetail.js

```diff
@@ -115,7 +115,7 @@ class GameDetail extends Component {
   };
 
   fetchCharacters = () => {
-    return this.props.portalStore.fetchProfileCharacters(this.props.portalStore.currentUser.profileID);
+    return this.props.portalStore.fetchProfileCharacters(this.props.portalStore.currentUser.profileID,0);
```

Comment:
Missing space after comma. Should be changed to `false` according to the other comments.